src/sentry/utils/samples.py

```diff
@@ -237,6 +239,16 @@ def load_data(
                     }
             measurements.update(measurement_markers)
 
+        if fingerprint is not None:
+            for f in fingerprint:
+                f_data = f.split("-")
+                if len(f_data) < 2:
+                    raise ValidationError(
+                        "Invalid performance fingerprint data. The format is: 'group_type-fingerprint'"
```

Comment:
this is meant to warn users when fingerprint is missing either a group type or the actual fingerprint